src/graph/graph.c

```diff
@@ -1383,6 +1400,17 @@ GrB_Matrix Graph_GetRelationMatrix(const Graph *g, int relation_idx) {
 	}
 }
 
+// Returns wether the matrix contains multi edge.
+bool Graph_RelationshipContainsMultiEdge(const Graph *g, int r) {
+	ASSERT(array_len(g->stats.edge_count) > r
+	&& array_len(g->relations) > r);
+	GrB_Index nvals;
+	GrB_Matrix_nvals(&nvals, RG_Matrix_Get_GrB_Matrix(g->relations[r]));
+
+	// A relationship matrix contains multi-edge if nvals < number of edges with type r.
```

Comment:
move to comment right above:
`GrB_Index nvals` this way it is clear why we're calling `GrB_Matrix_nvals`